src/gui/SettingsWidgetGeneral.ui

```diff
@@ -113,6 +113,16 @@
      </property>
     </widget>
    </item>
+   <item row="12" column="0">
+    <widget class="QCheckBox" name="systrayCloseToTrayCheckBox">
+     <property name="enabled">
+      <bool>false</bool>
+     </property>
+     <property name="text">
+      <string>Hide window to system tray when closing main window (Use Ctrl+Q or Database -> Exit to exit).</string>
```

Comment:
@kajzersoze, I see what you're getting at now.  This text will definitely need to change to match whatever the decided behavior is: it doesn't match even the patch it comes with.